src/Compilers/Core/Portable/Collections/ImmutableSegmentedDictionaryBuilderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Collections
+{
+    internal static class ImmutableSegmentedDictionaryBuilderExtensions
+    {
+        public static bool TryAdd<T>(
+            this ImmutableSegmentedDictionary<T, VoidResult>.Builder dictionary,
+            T value)
```

Comment:
Would it be appropriate to name this parameter 'key'?

Reply:
This is specifically implemented for treating a dictionary with `VoidResult` values as a set.